exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/Metadata.java

```diff
@@ -918,18 +916,22 @@ public void setMax(Object max) {
     @JsonProperty public ConcurrentHashMap<ColumnTypeMetadata_v2.Key, ColumnTypeMetadata_v2> columnTypeInfo;
     @JsonProperty List<ParquetFileMetadata_v2> files;
     @JsonProperty List<String> directories;
-    @JsonProperty String drillVersion;
```

Comment:
I had intentionally added the drill version here assuming that it would be good information to have around if a similar issue ever comes up the the future, as well as provide all of the info we need to have an explicit flag that the dates have become correct. For this to work completely, this commit should be the last commit right before a release (it could be a point release). Any particular reason that we would want to not write it into the file?